spec/javascripts/actions/story_spec.js

```diff
@@ -219,4 +223,130 @@ describe('Story Actions', () => {
       expect(fakeDispatch).toHaveBeenCalledWith(Story.updateHighlight(storyId, true));
     });
   });
+
+  describe('confirmBeforeSaveIfNeeded', () => {
+    let story;
+
+    beforeEach(() => {
+      story = storyFactory();
+    });
+
+    describe('when do not need of confirmation', () => {
+      const needConfirmation = sinon.stub().returns(false);
```

Comment:
We should move these to inside a `beforeEach`, let's avoid shared state between tests.